### PR TITLE
Add support of SAMPLE ClickHouse clause

### PR DESCRIPTION
**Pull request by YiuRULE**

### CODE REVIEW

**Comment by AzisK** [comment]
Could you pull from the master branch so that the newest CI test kick in?

**Comment by YiuRULE** [comment]
done, it has been rebase to master

File: pypika/dialects.py [hunk 1]
```diff
@@ -781,6 +781,16 @@ def drop_view(self, view: str) -> "ClickHouseDropQueryBuilder":
 class ClickHouseQueryBuilder(QueryBuilder):
     QUERY_CLS = ClickHouseQuery
 
+    def __init__(self, **kwargs) -> None:
+        super().__init__(self, as_keyword=True)
```
Comment (AzisK) [inline pypika/dialects.py:785]: is there no need to send other `kwargs` to the `super().__init__`?

Comment (YiuRULE) [reply pypika/dialects.py]: I think my previous code was indeed wrong, it has been corrected on the latest revision, thanks for bringing up the issue

**Review by AzisK — APPROVED** [review]
I have not had a chance to use Clickhouse in my life but the code looks good. Thanks for your work

**Pull request merged**